server/api/helpers/sermons/get-sermons.js

```diff
@@ -1,6 +1,22 @@
 const { DateTime } = require('luxon');
 const he = require('he');
 
+// removes html elements and attempts to remove sermon series from title
+// e.g. Sermon Series - Part 1: Sermon Title -> Part 1: Sermon Title
+// e.g. Sermon Series - Part 1 -> Part 1
+// e.g. Encounter: Sermon Title -> Encounter: Sermon Title
+const parseSermonTitle = (title) => {
+  // eslint-disable-next-line eqeqeq
+  if (title == null) return '';
+
+  const decodedTitle = he.decode(title);
+  const pattern = /.*(Part \d+.*)/g;
+  const captured = pattern.exec(decodedTitle);
+
+  // eslint-disable-next-line eqeqeq
+  return captured != null && captured[1] != null && captured[1] !== '' ? captured[1] : decodedTitle;
```

Comment:
if we edit in the api like this, then the title in this page, the title will just be "Part 3: Joy", I think we should display the full title in this case
![image](https://github.com/hmcc-global/hmcchk-web/assets/24607458/fea18f77-0f7e-45ee-8126-90c77dd37d12)

Reply:
so i can think of 2 ways to do this:
- add an argument in the API to return the parsed title or the full title
- add the 'series' back into the title in this page, but i think this is hacky

Reply:
i think if others agree with this suggestion, the solution is just to add a new field `fullTitle` or something. so card will use `sermon.title` while sermon pages will use `sermon.fullTitle`